volontulo/frontend/scss/bootstrap/_alerts.scss

```diff
@@ -68,6 +68,6 @@
   @include alert-variant($alert-warning-bg, $alert-warning-border, $alert-warning-text);
 }
 
-.alert-error {
+.alert-danger {
```

Comment:
In my opinion we shouldn't overwrite boostrap files becouse it's version can change and then we lose or changes. We should have custom - non bootstrap file with customizations.